Wikipedia/Code/TalkPageTopicReplyOnboardingView.swift

```diff
@@ -8,6 +8,16 @@ struct TalkPageTopicReplyOnboardingView: View {
     enum LocalizedStrings {
         static let title = WMFLocalizedString("talk-pages-topic-reply-onboarding-title", value: "Talk pages", comment: "Title of user education onboarding view for user and article talk pages.")
         static let body = WMFLocalizedString("talk-pages-topic-reply-onboarding-body", value: "Talk pages are where people discuss how to make content on Wikipedia the best that it can be. Add a new discussion topic to connect and collaborate with a community of Wikipedians.\n\nPlease be kind, we are all humans here.", comment: "Body text for user education onboarding view for user and article talk pages. Please do not translate the \n\n characters as these indicate where to insert new lines.")
+        @available(iOS 15, *)
+        static var bodyiOS15: AttributedString? = {
+            let localizedString = WMFLocalizedString("talk-pages-topic-reply-onboarding-body-ios15", value: "Talk pages are where %1$@people discuss how to make content on Wikipedia the best that it can be.%1$@ Add a new discussion topic to connect and collaborate with a community of Wikipedians.\n\nPlease be kind, we are all humans here.", comment: "Body text for user education onboarding view for user and article talk pages. Please do not translate the \n\n characters as these indicate where to insert new lines. The parameters also do not require translation, as they are used for markdown formatting. Parameters:\n* %1$@ - app-specific non-text formatting")
```

Comment:
One note about the `comment` here: the `Please do not translate the \n\n characters as these indicate where to insert new lines.` sentence can be removed. I was originally unaware but later told that the new lines characters just display as actual new lines on Translatewiki, so that comment to translators isn't necessary.

Reply:
Thanks for letting me know! I updated the comments